ui/src/main/webapp/src/app/shared/upload/alternative-upload-queue.component.scss

```diff
@@ -26,6 +26,7 @@ li {
 .progress-bar {
   padding-top: 5px;
   padding-bottom: 5px;
+  color: #363636;
```

Comment:
I agree with you that the `#363636` maybe is not the best color but if you look at the next screenshot, readability between the first two rows is quite the same, while between the forth and fifth is pretty much better the last because the previous white color is unreadable.

![screenshot from 2017-06-13 19-41-31](https://user-images.githubusercontent.com/7288588/27096338-04d6011a-5071-11e7-8ac9-57c5eb98be0a.png)

For you is it worse to read both on blue (uploading) and green (uploaded) background?